e2e/pages/enterApplicant/enterApplicant.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
     I.fillField(this.fields.applicant.name, applicant.name);
     I.fillField(this.fields.applicant.nameOfPersonToContact, applicant.nameOfPersonToContact);
     I.fillField(this.fields.applicant.jobTitle, applicant.jobTitle);
-    I.fillField(this.fields.applicant.address, applicant.address);
+    postcodeLookup.enterAddressManually(applicant);
```

Comment:
You're passing in a whole applicant but expecting an address in the method, can we rework the naming a little?

Reply:
As in just passing in applicant.address and removing the de-structuring in postcodeLookup frag. I like that better

Reply:
So I'm not fussed about passing in a whole applicant object, but just when using this enterAddressMethod it's telling me I need to give an address. Isn't intuitive, might be confusing to use later on

Reply:
I think using destructor and expecting someone to pass anything with an address is a bit confusing @Jamesnelson117. I would prefer much more having method that accepts exactly what is needed not anything that contains required bit fo data.

May I ask you to change it?